client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableStyles.ts

```diff
@@ -130,9 +130,9 @@ const useStyles = (isWide: boolean) => makeStyles((theme: Theme) => ({
     },
     searchBar: {
         justifyContent: 'flex-end',
-        width: '27vw',
+        width: '350px',
```

Comment:
why did you use px?

Reply:
the search bar needs to be a constant size and not dependent on the size's screen/font size 
- that's what the whole bug is about